Dockerfile

```diff
@@ -49,29 +50,40 @@ RUN curl -L http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz > /tmp/nginx
     cd /tmp && \
     tar zxf nginx-${NGINX_VERSION}.tar.gz
 
+
```

Comment:
Fixed in https://github.com/wantedly/nginx-image-server/pull/44/commits/6af385f5e4a0b9b914fe04644ff66f5b0e5f4f65.